compiler/modules/nback/build.bal

```diff
@@ -91,7 +91,7 @@ final RuntimeFunction taggedToIntFunction = {
         returnType: "i64",
         paramTypes: [LLVM_TAGGED_PTR]
     },
-    attrs: ["readonly"]
+    attrs: [[0, "readonly"]]
```

Comment:
This is rather ugly. I don't yet understand the problem this is solving, but I hope we can find something nicer.